### DISCUSSION TITLE
Getting dag_run inside pod_mutation_hook

### DISCUSSION THREAD

**Discussion by stefanondisponibile** [apache/airflow#19643] [Q&A]
Is it possible to get the current `dag_run` inside the `pod_mutation_hook`?
Specifically, is it possible to access the `dag_run.conf`?

---

Use case:

I want to modify the Pod configuration depending on some values passed from the _Trigger dag w/config_ .

**Comment by hterik** [comment]
See also https://github.com/apache/airflow/issues/19673
